### PR TITLE
add nutstash-wallet app

### PR DESCRIPTION
**Pull request by aphex3k**
@gandlafbtc This is what a PR for adding the nutstash wallet to umbrel app store would look like.

### CODE REVIEW

File: nutstash-wallet/umbrel-app.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  This app is in early development and most likely buggy. It does handle real sats though, so be sure to use only amounts that are small enough to lose.
+  
+  What is Cashu?
+
+  "Cashu is a free and open-source Chaumian ecash system built for Bitcoin. Cashu offers near-perfect privacy for users of custodial Bitcoin applications. Nobody needs to knows who you are, how much funds you have, and with whom you transact with." - cashu.space
+  
+releaseNotes: >-
+  - Initial release on the umbrel app store
+  
+developer: Joost Jager
```
Comment (gandlafbtc) [inline nutstash-wallet/umbrel-app.yml:17]: develper should be gandlafbtc, right?

Comment (aphex3k) [reply nutstash-wallet/umbrel-app.yml]: of course

File: nutstash-wallet/umbrel-app.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  This app is in early development and most likely buggy. It does handle real sats though, so be sure to use only amounts that are small enough to lose.
+  
+  What is Cashu?
+
+  "Cashu is a free and open-source Chaumian ecash system built for Bitcoin. Cashu offers near-perfect privacy for users of custodial Bitcoin applications. Nobody needs to knows who you are, how much funds you have, and with whom you transact with." - cashu.space
+  
+releaseNotes: >-
+  - Initial release on the umbrel app store
+  
+developer: Gandlabtc
```
Comment (gandlafbtc) [inline nutstash-wallet/umbrel-app.yml:17]: typo 😺 Gandlabtc -> gandlafbtc

**Comment by gandlafbtc** [comment]
i forked the umbrel-apps repo, if you want me to open the pr you can PR onto my repo and i will open a pr onto theirs

**Comment by nmfretz** [comment]
Looking forward to seeing this!

**Comment by gandlafbtc** [comment]
@nmfretz i will open the PR from my account

**Pull request closed without merging**